gym/envs/__init__.py

```diff
@@ -21,6 +21,13 @@
     max_episode_steps=200,
 )
 
+register(
+    id="Shoe-v0",
```

Comment:
Could you edit `README.md` as well and add `Shoe-v0` to the list of supported list of environments?